packages/react-provider/src/reducer.ts

```diff
@@ -29,11 +29,12 @@ const rootReducer = (state, action) => {
       return {
         ...state,
         apiUrl: action.payload.apiUrl,
+        brandId: action.payload.brandId,
         clientKey: action.payload.clientKey,
+        graphQLClient: action.payload.graphQLClient,
```

Comment:
curious why we need graphQLClient here if its available on context?

Reply:
because it can changed based on the clientKey and userId